### PR TITLE
Issue 5: Add Image Tags

### PR DESCRIPTION
**Pull request by abhishekupasani24**
Closes #5

### CODE REVIEW

**Comment by abhishekupasani24** [comment]
[Issue 5: Image Tags](https://trello.com/c/BxMHyJrt/234-issue-5-image-tags)

File: app/controllers/images_controller.rb [hunk 1]
```diff
@@ -24,6 +24,6 @@ def create
   private
 
   def image_params
-    params.require(:image).permit(:url)
+    params.require(:image).permit(:url, :tag_list)
```
Comment (MMullin7) [inline app/controllers/images_controller.rb]: so we're including new data here, we should probably have a test somewhere that takes that information and make sure its used correctly.

File: test/models/image_test.rb [hunk 2]
```diff
@@ -15,4 +15,10 @@ class ImageTest < ActiveSupport::TestCase
     image = Image.new(url: '~123fdsaasd(')
     refute_predicate image, :valid?
   end
+
+  test 'image valid when tag added' do
```
Comment (MMullin7) [inline test/models/image_test.rb]: I don't think this test is necessary.  valid? is a check against model validation. The acts on taggable  addition to the model has no validations for your model.  So I think we can just delete this.  
HOWEVER: If you added a new validation about max tags hypothetically then a test would be needed

File: app/controllers/images_controller.rb (hunk 1 above)
Comment (abhishekupasani24) [reply app/controllers/images_controller.rb]: Done

File: test/models/image_test.rb (hunk 2 above)
Comment (abhishekupasani24) [reply test/models/image_test.rb]: SUre

**Pull request merged**